util/crontab-example

```diff
@@ -0,0 +1,2 @@
+SHELL=/bin/bash
+0 7 * * * cd $HOME/wptdashboard && ./run/run.py firefox-56.0-linux --upload --create-testrun &>> /var/log/wptd.log
```

Comment:
This file is simply to help me (and others who may help maintain it) remember the cron command for running it in the VMs.

The tests are currently being run this way, and I have a bug up (#42) for discussing long-term if we do want to move to LUCI or if we want to move toward something less Chromium-specific.